elide-core/src/main/java/com/yahoo/elide/jsonapi/EntityProjectionMaker.java

```diff
@@ -136,11 +138,16 @@ public EntityProjection parseInclude(Class<?> entityClass) {
             Class<?> entityClass = getEntityClass(parentClass, entityName);
             FilterExpression filter = scope.getExpressionForRelation(parentClass, entityName).orElse(null);
 
+            Sorting sorting = SortingImpl.getDefaultEmptyInstance();
+            if (scope.getQueryParams().isPresent()) {
+                sorting = SortingImpl.parseQueryParams(scope.getQueryParams().get(), entityClass, dictionary);
```

Comment:
Could this be simplified with:
```java
Sorting sorting = SortingImpl.parseQueryParams(scope.getQueryParams().get(), entityClass, dictionary);
```

Reply:
That doesn't cover the case where there are no query params.   I believe the code is as simple as can be.

Reply:
Can this cover that case:
https://github.com/yahoo/elide/pull/1146/files#diff-9f7716d84a63e43ad20f499737c723cdR152

Reply:
Check out the new refactor.